src/components/templates/Product/Card.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+
+import { Card, breakpoints, Heading, Text, colors, useViewport } from '../../../index';
+
+interface ProductTemplateCardProps {
+  notification?: ReactNode;
+  title?: ReactNode;
+  subtitle?: ReactNode;
```

Comment:
Do we need to specify props for these, could they not be achieved by passing them in as part of `children` giving teams more flexibility

Reply:
Yeah thats fair

Reply:
I think the point of the template page is to make it the less flexible as possible. If we start giving flexibility what's the point of the template?

Reply:
I agree, we should make it less flexible for layout, header etc but this for the content area. We should be able to put in any content we like. We already accept children here so I can see teams ignoring the title, subtitle and notification props and just passing in what content they want to show

Reply:
Ahh yes, make sense for the content area. Agree